packages/protocol/contracts/L2/TaikoL2.sol

```diff
@@ -215,14 +215,10 @@ contract TaikoL2 is EssentialContract, TaikoL2Signer, IXchainSync {
      **********************/
 
     function getBasefee(
-        uint32 timeSinceNow,
+        uint32 timeSinceParent,
```

Comment:
@davidtaikocha Lets talk about it, I think you may missed something here.

Reply:
lets talk about it tomorrow.